Extensions/Optimizely/Src/Tasks/OptimizelyX/package.json

```diff
@@ -9,10 +9,10 @@
   "author": "Microsoft Corporation",
   "license": "MIT",
   "dependencies": {
-    "@types/mocha": "2.2.48",
```

Comment:
Thus @type/node or any other global-inducing type definitions should be in devDependencies